client/src/views/AdminView/AdminView.jsx

```diff
@@ -53,7 +54,8 @@ const AdminView = () => {
       )
 
       if (response.status === 200) {
-        navigate('/success') // Navigate to success page on successful invite
+        // navigate('/success') // Navigate to success page on successful invite
+        setSuccessMessage('Invited user succesfully!')
```

Comment:
I think this message should be something more along the lines of
`"Instructor invitations sent successfully"`